tensorvision/utils.py

```diff
@@ -56,14 +56,18 @@ def set_dirs(hypes, hypes_fname):
         else:
             runs_dir = os.path.join(base_path, 'RUNS')
 
+        # test for project dir
+        if hasattr(FLAGS, 'project') and FLAGS.project is not None:
+            runs_dir = os.path.join(runs_dir, FLAGS.project)
+
```

Comment:
Well, if --project is not set, it will be saved to '$TV_DIR_RUNS'. This seems to be convenient for the standard case.